test/classes/deitz/types/type_to_value_error.good

```diff
@@ -1 +1,4 @@
+type_to_value_error.chpl:7: warning: unnecessary type construction call
```

Comment:
I'd expect this and the test below to also have been impacted by the change you just made to resolution for `class1-old.chpl`.  Did you do a full paratest after making that change?

Reply:
I did run a full paratest. Which test is the one you meant by "the test below"? I can adjust the pattern matching to hide the new warning in this case, but right now the pattern matching is too specific to variable declaration with default-init to find it.

Reply:
throwNew.chpl

Reply:
OK, I've adjusted the logic so that the warning is skipped for these two tests.